### DISCUSSION TITLE
SEMPRE APARECE OS MESMOS VIDEOS

### DISCUSSION THREAD

**Discussion by flavio085** [aclap-dev/video-downloadhelper#843] [Help and Q&A]
### Your question

Sempre fica aparecendo os mesmos videos, por favor, me ajudem!
Como devo fazer para aparecer novos videos???????

AQUELE MANUAL DE AJUDA NAO SERVE PRA NADA.

<img width="310" alt="SEMPRE-APARECE-OS-MESMOS-VIDEOS" src="https://github.com/aclap-dev/video-downloadhelper/assets/171888207/9d6d17d6-0b67-4c12-b4e5-0142cb8dc7a2">

### Addon & CoApp version and details

Sempre fica aparecendo os mesmos videos, por favor, me ajudem!
Como devo fazer para aparecer novos videos???????

AQUELE MANUAL DE AJUDA NAO SERVE PRA NADA.

<img width="310" alt="SEMPRE-APARECE-OS-MESMOS-VIDEOS" src="https://github.com/aclap-dev/video-downloadhelper/assets/171888207/48834033-9b44-4340-a567-a275afccffd5">

**Comment by paulrouget** [comment]
In the menu, under the 3 dots, use the blacklist function.
